### ISSUE TITLE
Add a copy to clipboard button

### ISSUE THREAD

**Issue by mrjoelkemp** [mrjoelkemp/linklink#6]
##

**Comment by dandv** [comment]
ZeroClipboard works pretty well.

**Comment by mrjoelkemp** [comment]
Agreed. I've used it in the past. I consciously tried to keep the number of 3rd party libs being loaded to a minimum for both speed and to avoid conflicting with the loaded site.

I'll likely use ZeroClipboard though. Thanks for the suggestion!

**Comment by dandv** [comment]
If you're trying to prioritize work, it might be worth to wait until ZeroClipboard fixes [this issue](https://github.com/zeroclipboard/zeroclipboard/issues/171).
